templates/injector-deployment.yaml

```
@@ -109,6 +109,8 @@ spec:
               value: "{{ .Values.injector.agentDefaults.template }}"
             - name: AGENT_INJECT_TEMPLATE_CONFIG_EXIT_ON_RETRY_FAILURE
               value: "{{ .Values.injector.agentDefaults.templateConfig.exitOnRetryFailure }}"
+            - name: AGENT_INJECT_TEMPLATE_CONFIG_STATIC_SECRET_RENDER_INTERVAL
```

I believe this environment variable was shortened a bit:

```suggestion
            - name: AGENT_INJECT_TEMPLATE_STATIC_SECRET_RENDER_INTERVAL
```